stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -292,6 +685,11 @@ protected void onUnmapped(Object object, int id) {
 
   private final class ProviderListener implements DOMProvider.Listener {
     @Override
+    public void onChanged() {
+      updateTree();
```

Comment:
Wrap this with SystemClock timings and print the result using `LogUtil.d`.  It'd be useful for us to track this easily and also highlight to interested developers how this works.  Might also keep us honest about improving it :)